### PR TITLE
App strings reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -36,40 +36,42 @@
...
 	<string name="automatic">Automatic</string>
 	<string name="pages">Pages</string>
 	<string name="clear">Clear</string>
-	<string name="text_clear_history_prompt">Do you really want to clear all your reading history? This action cannot be undone.</string>
+	<string name="text_clear_history_prompt">Clear all reading history permanently?</string>
 	<string name="remove">Remove</string>
 	<string name="_s_removed_from_history">\"%s\" removed from history</string>
 	<string name="_s_deleted_from_local_storage">\"%s\" deleted from local storage</string>
-	<string name="wait_for_loading_finish">Wait for the load to finish</string>
+	<string name="wait_for_loading_finish">Waiting for loading to finish…</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:64]: What happens here?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -78,146 +80,144 @@
...
 	<string name="report_github">Create issue on GitHub</string>
 	<string name="scale_mode">Scale mode</string>
 	<string name="zoom_mode_fit_center">Fit center</string>
 	<string name="zoom_mode_fit_height">Fit to height</string>
 	<string name="zoom_mode_fit_width">Fit to width</string>
 	<string name="zoom_mode_keep_start">Keep at start</string>
-	<string name="black_dark_theme">Black dark theme</string>
-	<string name="black_dark_theme_summary">Useful for AMOLED screens</string>
-	<string name="restart_required">Restart required</string>
+	<string name="restart_required">App restart required</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:178]: Assuming this should not be "system restart"

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -78,146 +80,144 @@
...
 	<string name="internal_storage">Internal storage</string>
 	<string name="external_storage">External storage</string>
 	<string name="domain">Domain</string>
-	<string name="application_update">Check for updates automatically</string>
-	<string name="app_update_available">Application update is available</string>
-	<string name="show_notification_app_update">Show notification if update is available</string>
+	<string name="application_update">Check for new versions of the app</string>
+	<string name="app_update_available">A new version of the app is available</string>
+	<string name="show_notification_app_update">Show notification if a new version is available</string>
 	<string name="open_in_browser">Open in browser</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:104]: Web or file browser?

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -78,146 +80,144 @@
...
 	<string name="checking_for_updates">Checking for updates…</string>
-	<string name="update_check_failed">Update check failed</string>
+	<string name="update_check_failed">Could not look for updates</string>
 	<string name="no_update_available">No updates available</string>
-	<string name="right_to_left">Right to left</string>
-	<string name="prefer_rtl_reader">Prefer Right to left reader</string>
-	<string name="prefer_rtl_reader_summary">You can set up the reading mode for each manga separately</string>
+	<string name="right_to_left">Right-to-left ←</string>
+	<string name="prefer_rtl_reader">Prefer right-to-left reader</string>
+	<string name="prefer_rtl_reader_summary">Reading mode can be set up separately for each file</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:170]:
```suggestion
	<string name="prefer_rtl_reader_summary">Reading mode can be set up separately for each series</string>
```
?

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -78,146 +80,144 @@
...
+	<string name="data_restored_with_errors">The data restored, but with errors</string>
+	<string name="backup_information">Create a backup of your history and favourites to be restored later</string>
 	<string name="just_now">Just now</string>
 	<string name="yesterday">Yesterday</string>
 	<string name="long_ago">Long ago</string>
 	<string name="group">Group</string>
 	<string name="today">Today</string>
 	<string name="tap_to_try_again">Tap to try again</string>
-	<string name="reader_mode_hint">Chosen configuration will be remembered for this manga</string>
+	<string name="reader_mode_hint">Chosen configuration will be remembered for this file</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:194]:
```suggestion
	<string name="reader_mode_hint">Chosen configuration will be remembered for this series</string>
```
?

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (nv95) [reply app/src/main/res/values/strings.xml:64]: This message showing if user click on some of buttons in reader while content is not loaded yet. I think, this string should be removed

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (nv95) [reply app/src/main/res/values/strings.xml:104]: Web

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -22,12 +22,12 @@
...
 	<string name="nothing_found">Nothing found</string>
-	<string name="history_is_empty">History is empty</string>
+	<string name="history_is_empty">No history yet</string>
 	<string name="read">Read</string>
 	<string name="add_bookmark">Add bookmark</string>
-	<string name="you_have_not_favourites_yet">You have not favourites yet</string>
-	<string name="add_to_favourites">Add to favourites</string>
-	<string name="add_new_category">Add new category</string>
+	<string name="you_have_not_favourites_yet">No favourites yet</string>
+	<string name="add_to_favourites">Favourite</string>
```
Comment (ztimms73) [inline app/src/main/res/values/strings.xml:29]: It is not quite clear whether this is an action or not?

File: app/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -36,40 +36,42 @@
...
 	<string name="all">All</string>
-	<string name="sort_order">Sort order</string>
+	<string name="sort_order">Sorting order</string>
 	<string name="genre">Genre</string>
 	<string name="filter">Filter</string>
 	<string name="theme">Theme</string>
 	<string name="light">Light</string>
 	<string name="dark">Dark</string>
+	<string name="black_dark_theme">Black</string>
+	<string name="black_dark_theme_summary">Uses less power on AMOLED screens</string>
```
Comment (ztimms73) [inline app/src/main/res/values/strings.xml:56]: This is a myth. https://www.xda-developers.com/amoled-black-vs-gray-dark-mode/

File: app/src/main/res/values/strings.xml [hunk 8]
```diff
@@ -36,40 +36,42 @@
...
+	<string name="by_rating">Rating</string>
 	<string name="all">All</string>
-	<string name="sort_order">Sort order</string>
+	<string name="sort_order">Sorting order</string>
 	<string name="genre">Genre</string>
 	<string name="filter">Filter</string>
 	<string name="theme">Theme</string>
 	<string name="light">Light</string>
 	<string name="dark">Dark</string>
+	<string name="black_dark_theme">Black</string>
```
Comment (ztimms73) [inline app/src/main/res/values/strings.xml:55]: Black what?

File: app/src/main/res/values/strings.xml [hunk 9]
```diff
@@ -36,40 +36,42 @@
...
-	<string name="text_clear_history_prompt">Do you really want to clear all your reading history? This action cannot be undone.</string>
+	<string name="text_clear_history_prompt">Clear all reading history permanently?</string>
 	<string name="remove">Remove</string>
 	<string name="_s_removed_from_history">\"%s\" removed from history</string>
 	<string name="_s_deleted_from_local_storage">\"%s\" deleted from local storage</string>
-	<string name="wait_for_loading_finish">Wait for the load to finish</string>
+	<string name="wait_for_loading_finish">Waiting for loading to finish…</string>
 	<string name="save_page">Save page</string>
-	<string name="page_saved">Page saved successful</string>
+	<string name="page_saved">Saved</string>
```
Comment (ztimms73) [inline app/src/main/res/values/strings.xml:66]: What saved?

File: app/src/main/res/values/strings.xml (hunk 8 above)
Comment (nv95) [reply app/src/main/res/values/strings.xml:55]: This string used as a preference title, so just "Black" is not clear to understand

File: app/src/main/res/values/strings.xml (hunk 9 above)
Comment (comradekingu) [reply app/src/main/res/values/strings.xml:66]: The page. Evident from both the above, and the key context.

File: app/src/main/res/values/strings.xml (hunk 8 above)
Comment (comradekingu) [reply app/src/main/res/values/strings.xml:55]: It is black, and that is what I've seen other apps use.
Some have a less than 100% black option called "dark".

File: app/src/main/res/values/strings.xml (hunk 7 above)
Comment (comradekingu) [reply app/src/main/res/values/strings.xml:56]: It is not. (The headline is clickbait, and openly contradicted in the article) What I think you are confusing for this white vs black statement is whether black uses less power than gray.
The later is true, but it is not very meaningful, and the bleed and glare as perceived by the eye is a lot more important (in favour of dark).
In Kotatsu, "dark" is already black, and the switch thereby doesn't really do anything(?).

TL;DR the statement is true, and now a bit easier to read.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (comradekingu) [reply app/src/main/res/values/strings.xml:64]: OK

**Comment by comradekingu** [comment]
@nv95 @ztimms73 
Good to go?

**Comment by comradekingu** [comment]
@nv95 @ztimms73 Good to go?

File: app/src/main/res/values/strings.xml [hunk 10]
```diff
@@ -7,8 +7,8 @@
     <string name="local_storage">Local storage</string>
     <string name="favourites">Favourites</string>
     <string name="history">History</string>
-    <string name="error_occurred">An error has occurred</string>
-    <string name="network_error">Network connection error</string>
+    <string name="error_occurred">An error occurred</string>
+    <string name="network_error">Could not connect to the Internet</string>
```
Comment (nv95) [inline app/src/main/res/values/strings.xml:11]: Not sure that it is correct

File: app/src/main/res/values/strings.xml [hunk 11]
```diff
@@ -37,40 +37,40 @@
...
     <string name="all">All</string>
-    <string name="sort_order">Sort order</string>
+    <string name="sort_order">Sorting order</string>
     <string name="genre">Genre</string>
     <string name="filter">Filter</string>
     <string name="theme">Theme</string>
     <string name="light">Light</string>
     <string name="dark">Dark</string>
-    <string name="automatic">Automatic</string>
+    <string name="automatic">System</string>
```
Comment (nv95) [inline app/src/main/res/values/strings.xml:56]:
```suggestion
    <string name="automatic">Follow system</string>
```

File: app/src/main/res/values/strings.xml [hunk 12]
```diff
@@ -37,40 +37,40 @@
...
-    <string name="text_clear_history_prompt">Do you really want to clear all your reading history? This action cannot be undone.</string>
+    <string name="text_clear_history_prompt">Clear all reading history permanently?</string>
     <string name="remove">Remove</string>
     <string name="_s_removed_from_history">\"%s\" removed from history</string>
     <string name="_s_deleted_from_local_storage">\"%s\" deleted from local storage</string>
-    <string name="wait_for_loading_finish">Wait for the load to finish</string>
+    <string name="wait_for_loading_finish">Wait for loading to finish…</string>
     <string name="save_page">Save page</string>
-    <string name="page_saved">Page saved successful</string>
+    <string name="page_saved">Saved</string>
```
Comment (nv95) [inline app/src/main/res/values/strings.xml:65]: User should understand, _what_ saved

File: app/src/main/res/values/strings.xml [hunk 13]
```diff
@@ -79,174 +79,174 @@
     <string name="grid_size">Grid size</string>
     <string name="search_on_s">Search on %s</string>
     <string name="delete_manga">Delete manga</string>
-    <string name="text_delete_local_manga">Do you really want to delete \"%s\" from your phone\'s local storage? \nThis operation cannot be undone.</string>
+    <string name="text_delete_local_manga">Delete \"%s\" from device permanently?</string>
```
Comment (nv95) [inline app/src/main/res/values/strings.xml:82]: Too short messages, it can be more informative

File: app/src/main/res/values/strings.xml [hunk 14]
```diff
@@ -79,174 +79,174 @@
...
     <string name="reader_settings">Reader settings</string>
     <string name="switch_pages">Switch pages</string>
-    <string name="taps_on_edges">Taps on edges</string>
+    <string name="taps_on_edges">Edge taps</string>
     <string name="volume_buttons">Volume buttons</string>
     <string name="_continue">Continue</string>
     <string name="warning">Warning</string>
-    <string name="network_consumption_warning">This operation may consume a lot of network traffic</string>
+    <string name="network_consumption_warning">This may download a lot of power</string>
```
Comment (nv95) [inline app/src/main/res/values/strings.xml:89]: The meaning is missed

File: app/src/main/res/values/strings.xml [hunk 15]
```diff
@@ -79,174 +79,174 @@
...
     <string name="dont_ask_again">Don\'t ask again</string>
     <string name="cancelling_">Cancelling…</string>
     <string name="error">Error</string>
     <string name="clear_thumbs_cache">Clear thumbnails cache</string>
     <string name="clear_search_history">Clear search history</string>
-    <string name="search_history_cleared">Search history cleared</string>
+    <string name="search_history_cleared">Cleared</string>
```
Comment (nv95) [inline app/src/main/res/values/strings.xml:95]: _What_ cleared?

File: app/src/main/res/values/strings.xml [hunk 16]
```diff
@@ -79,174 +79,174 @@
...
+    <string name="light_indicator">LED indicator</string>
     <string name="vibration">Vibration</string>
-    <string name="favourites_categories">Favourites categories</string>
+    <string name="favourites_categories">Favourite categories</string>
     <string name="categories_">Categories…</string>
     <string name="rename">Rename</string>
-    <string name="category_delete_confirm">Do you really want to remove category \"%s\" from your favourites? \nAll containing manga will be lost.</string>
-    <string name="remove_category">Remove category</string>
+    <string name="category_delete_confirm">Remove the \"%s\" category from your favourites? \nAll manga in it will be lost.</string>
+    <string name="remove_category">Remove</string>
```
Comment (nv95) [inline app/src/main/res/values/strings.xml:121]: Remove _what_

File: app/src/main/res/values/strings.xml [hunk 17]
```diff
@@ -79,174 +79,174 @@
...
-    <string name="manga_shelf">Manga shelf</string>
-    <string name="recent_manga">Recent manga</string>
-    <string name="pages_animation">Pages animation</string>
-    <string name="manga_save_location">Manga download location</string>
+    <string name="text_history_holder_primary">What you read will be displayed here</string>
+    <string name="text_history_holder_secondary">Find what to read in side menu.</string>
+    <string name="text_local_holder_primary">Save something first</string>
+    <string name="text_local_holder_secondary">Save it from online sources or import files.</string>
+    <string name="manga_shelf">Shelf</string>
+    <string name="recent_manga">Recent</string>
```
Comment (nv95) [inline app/src/main/res/values/strings.xml:130]: Recent _what_? This strings are used for widgets

File: app/src/main/res/values/strings.xml [hunk 18]
```diff
@@ -79,174 +79,174 @@
...
+    <string name="pages_animation">Page animation</string>
+    <string name="manga_save_location">Downloads</string>
```
Comment (nv95) [inline app/src/main/res/values/strings.xml:132]: Absolutely not informative

File: app/src/main/res/values/strings.xml [hunk 19]
```diff
@@ -79,174 +79,174 @@
...
-    <string name="text_feed_holder">Here you will see the new chapters of the manga you are reading</string>
+    <string name="text_feed_holder">New chapters of what you are reading is shown here</string>
     <string name="search_results">Search results</string>
     <string name="related">Related</string>
     <string name="new_version_s">New version: %s</string>
     <string name="size_s">Size: %s</string>
     <string name="waiting_for_network">Waiting for network…</string>
     <string name="clear_updates_feed">Clear updates feed</string>
-    <string name="updates_feed_cleared">Updates feed cleared</string>
+    <string name="updates_feed_cleared">Cleared</string>
```
Comment (nv95) [inline app/src/main/res/values/strings.xml:149]: Cleared _what_?

File: app/src/main/res/values/strings.xml [hunk 20]
```diff
@@ -79,174 +79,174 @@
...
+    <string name="prefer_rtl_reader_summary">Reading mode can be set up separately for each series</string>
     <string name="create_category">New category</string>
     <string name="report_github">Create issue on GitHub</string>
     <string name="scale_mode">Scale mode</string>
     <string name="zoom_mode_fit_center">Fit center</string>
     <string name="zoom_mode_fit_height">Fit to height</string>
     <string name="zoom_mode_fit_width">Fit to width</string>
     <string name="zoom_mode_keep_start">Keep at start</string>
-    <string name="black_dark_theme">Black dark theme</string>
+    <string name="black_dark_theme">Black</string>
```
Comment (nv95) [inline app/src/main/res/values/strings.xml:177]: Black _what_?

File: app/src/main/res/values/strings.xml [hunk 21]
```diff
@@ -79,174 +79,174 @@
...
     <string name="black_dark_theme_summary">Useful for AMOLED screens</string>
     <string name="restart_required">Restart required</string>
-    <string name="backup_restore"><![CDATA[Backup & Restore]]></string>
+    <string name="backup_restore"><![CDATA[Backup and restore]]></string>
     <string name="create_backup">Create data backup</string>
     <string name="restore_backup">Restore from backup</string>
-    <string name="data_restored">Data restored</string>
+    <string name="data_restored">Restored</string>
```
Comment (nv95) [inline app/src/main/res/values/strings.xml:183]: Restored _what_

File: app/src/main/res/values/strings.xml [hunk 22]
```diff
@@ -79,174 +79,174 @@
...
-    <string name="text_clear_search_history_prompt">Do you really want to remove all recent search queries? This action cannot be undone.</string>
+    <string name="text_clear_search_history_prompt">Remove all recent search queries permanently?</string>
     <string name="other">Other</string>
     <string name="languages">Languages</string>
     <string name="welcome">Welcome</string>
     <string name="description">Description</string>
-    <string name="backup_saved">Backup saved successfully</string>
-    <string name="tracker_warning">Some manufacturers can change the system behavior, which may breaks background tasks.</string>
+    <string name="backup_saved">Backup saved</string>
+    <string name="tracker_warning">Some devices have different system behavior, which may break background tasks.</string>
```
Comment (nv95) [inline app/src/main/res/values/strings.xml:222]: I think original meaning is lost

File: app/src/main/res/values/strings.xml [hunk 23]
```diff
@@ -79,174 +79,174 @@
...
     <string name="genres">Genres</string>
     <string name="state_finished">Finished</string>
     <string name="state_ongoing">Ongoing</string>
     <string name="date_format">Date format</string>
     <string name="system_default">Default</string>
     <string name="exclude_nsfw_from_history">Exclude NSFW manga from history</string>
-    <string name="error_empty_name">Name should not be empty</string>
-    <string name="show_pages_numbers">Show pages numbers</string>
-    <string name="enabled_sources">Enabled sources</string>
+    <string name="error_empty_name">You must pick a name</string>
```
Comment (nv95) [inline app/src/main/res/values/strings.xml:248]: I think "pick" is a "select", here should be "type"

**Review by nv95 — COMMENTED** [review]
Thank you for your work!

A lot of useful fixes but some changes are disputable, not sure it should be merged as is. Of course, English is not my native language, so I cannot be sure that my opinion is absolutely correct)

File: app/src/main/res/values/strings.xml (hunk 10 above)
Comment (comradekingu) [reply app/src/main/res/values/strings.xml:11]: https://en.wikipedia.org/wiki/Internet is correct.
Ultimately the problem could be just the local network, but if the objective is to always communicate externally, it isn't untrue.

File: app/src/main/res/values/strings.xml (hunk 12 above)
Comment (comradekingu) [reply app/src/main/res/values/strings.xml:65]: I thought it should be obvious from "Save page".
For translators the key says "page_saved", so that is fine.

File: app/src/main/res/values/strings.xml (hunk 13 above)
Comment (comradekingu) [reply app/src/main/res/values/strings.xml:82]: It may not be a phone, but is always a device.
"local storage" seems technical for no reason.

File: app/src/main/res/values/strings.xml (hunk 14 above)
Comment (comradekingu) [reply app/src/main/res/values/strings.xml:89]: I have no idea what I was thinking here. Good catch.

Comment (comradekingu) [inline app/src/main/res/values/strings.xml:89]:
```suggestion
    <string name="network_consumption_warning">This may transfer a lot of data</string>
```

File: app/src/main/res/values/strings.xml (hunk 18 above)
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:132]:
```suggestion
    <string name="manga_save_location">Folder for downloads</string>
```

File: app/src/main/res/values/strings.xml (hunk 20 above)
Comment (comradekingu) [reply app/src/main/res/values/strings.xml:177]: Theme. It is always under that header.
Otherwise it is
```
Theme
Dark theme
Light theme
```
when selecting, and
```
Theme
Dark
```
in the menu.

File: app/src/main/res/values/strings.xml [hunk 24]
```diff
@@ -79,174 +79,174 @@
...
     <string name="create_category">New category</string>
     <string name="report_github">Create issue on GitHub</string>
     <string name="scale_mode">Scale mode</string>
     <string name="zoom_mode_fit_center">Fit center</string>
     <string name="zoom_mode_fit_height">Fit to height</string>
     <string name="zoom_mode_fit_width">Fit to width</string>
     <string name="zoom_mode_keep_start">Keep at start</string>
-    <string name="black_dark_theme">Black dark theme</string>
+    <string name="black_dark_theme">Black</string>
     <string name="black_dark_theme_summary">Useful for AMOLED screens</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:178]:
```suggestion
    <string name="black_dark_theme_summary">Uses less power on AMOLED screens</string>
```

File: app/src/main/res/values/strings.xml (hunk 22 above)
Comment (comradekingu) [reply app/src/main/res/values/strings.xml:222]: Could this be explained?
I was trying to not invoke "manufacturer", since I am guessing the real issue is the Android version, or some setup done by OEMs/vendors.

File: app/src/main/res/values/strings.xml (hunk 23 above)
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:248]:
```suggestion
    <string name="error_empty_name">You must enter a name</string>
```

File: app/src/main/res/values/strings.xml (hunk 22 above)
Comment (nv95) [reply app/src/main/res/values/strings.xml:222]: This is about OEM`s modifications

File: app/src/main/res/values/strings.xml (hunk 20 above)
Comment (nv95) [reply app/src/main/res/values/strings.xml:177]: Hm, ok

**Comment by nv95** [comment]
Ok, I think, it can be merged

@ztimms73 what you think about it?

**Comment by ztimms73** [comment]
> @ztimms73 what you think about it?

Yeah, I'm also think it's okay 👍
